scripts/update.py

```diff
@@ -0,0 +1,129 @@
+import json, shutil, subprocess
+
+from github import Github
+
+from releases import get_latest_releases, SAMPLES_REPO
+from util import get_path_from_version_id, get_version_parts, write_to_github_output
+from docs import get_docs_update
+
+from constants import Version, LINE, ROOT, TMP_PATH, TAGS_PATH, TITLES, IS_ACTIONS
+
+g = Github()
```

Comment:
What does this line do? `g` is a terrible name. Is it even used?

Reply:
Moved to a better location and renamed.